crates/federate/src/send.rs

```diff
@@ -66,7 +72,7 @@ pub(crate) enum SendActivityResult {
 /// to send an activity to multiple inbox URLs, with built-in retry logic.
 pub(crate) struct SendRetryTask<'a> {
   pub activity: &'a SentActivity,
-  pub object: &'a SharedInboxActivities,
+  pub object: &'a Value,
```

Comment:
would be good to add a comment here that this value is of type SharedInboxActivities but not deserialized into it to avoid a dependency

Reply:
Done